src/System.Net.WebSockets/ref/System.Net.WebSockets.cs

```diff
@@ -117,7 +117,7 @@ public partial class WebSocketReceiveResult
         public WebSocketReceiveResult(int count, System.Net.WebSockets.WebSocketMessageType messageType, bool endOfMessage, System.Nullable<System.Net.WebSockets.WebSocketCloseStatus> closeStatus, string closeStatusDescription) { }
         public System.Nullable<System.Net.WebSockets.WebSocketCloseStatus> CloseStatus { get { throw null; } }
         public string CloseStatusDescription { get { throw null; } }
-        public int Count { get { throw null; } }
+        public int Count { get { throw null; } set { } }
```

Comment:
We should not be adding APIs as part of this.

Reply:
I will change to follow the Framework's pattern.